docs/BesuProxyModel.md

```diff
@@ -0,0 +1,58 @@
+
+**Testing Avalon Proxy model with Hyperledger Besu**
+1.  Install ``truffle`` using the command
+    ```
+    npm install -g truffle
+    ```
+    This requires Requires ``NodeJS v8.9.4 or later`` preinstalled.
+
+2.  Set the environment variable ``TCF_HOME`` to the Avalon root directory
+
+3.  Start the Ethereum network with 2 nodes and 2 eth-signers. To bring up the network, do the following
+       ```
+    cd $TCF_HOME/docs/dev-environments/besu
+    docker-compose up -d
+    ```
+       This will start the Ethereum network locally.
+
+4.  Now you need to create a truffle project and deploy contracts onto the Ethereum network
+       ```
+        mkdir ./my_project
+        cd my_project
```

Comment:
Explain how to create a default truffle project. One way is to clone this repo -> https://github.com/trufflesuite/truffle-init-default